src/peer_connection.cpp

```diff
@@ -5616,7 +5616,7 @@ namespace libtorrent {
 #ifndef TORRENT_DISABLE_LOGGING
 		peer_log(peer_log_alert::outgoing, "ASYNC_WRITE", "bytes: %d", amount_to_send);
 #endif
-		std::vector<boost::asio::const_buffer> const& vec = m_send_buffer.build_iovec(amount_to_send);
+		auto const& vec = m_send_buffer.build_iovec(amount_to_send);
```

Comment:
Is it better to keep the `&` here or not?

Reply:
Oh, it should probably make a copy, but it probably doesn’t matter in practice, because of lifetime extension, and it being copied into the handler op